Content.Server/Speech/EntitySystems/PirateAccentSystem.cs

```diff
@@ -0,0 +1,108 @@
+using System.Globalization;
+using System.Text.RegularExpressions;
+using Content.Server.Speech.Components;
+
+namespace Content.Server.Speech.EntitySystems;
+
+public sealed class PirateAccentSystem : EntitySystem
+{
+    private static readonly Dictionary<string, string> DirectReplacements = new()
+    {
+        { "hi", "Arghh" },
+        { "hello", "Ahoy" },
+        { "hey", "Gar" },
+        { "yo", "Yarr" },
+        { "friend", "matey" },
+        { "crew", "me harties" },
```

Comment:
believe this should be "me hearties"

Reply:
yea will change it

Reply:
None of these are localised either.

Also isn't there a replacementaccentcomponent for straight up replacements?